### PR TITLE
Update icons

### PR DESCRIPTION
**Pull request by keithtxw**
**Changes**
Update icons to use the one provided by UX

- [delete] branch

**Changelog entry**

- [WARNING] Update `Icon` collection

### CODE REVIEW

File: src/notification-banner/notification-banner.styles.tsx [hunk 1]
```diff
@@ -19,11 +19,19 @@ interface WrapperStyleProps {
 // =============================================================================
```
Comment (weili-govtech) [inline src/notification-banner/notification-banner.styles.tsx:19]: is `notification-banner-label.tsx` still used / intended to be used? can either combine or remove that file

**Review by qroll — COMMENTED** [review]
redundant font-weight and font-size css applied on some `Styled(Icon)`, can be cleaned up

File: src/notification-banner/notification-banner.styles.tsx (hunk 1 above)
Comment (keithtxw) [reply src/notification-banner/notification-banner.styles.tsx:19]: nope it is no longer intended. the file is not present though. What do u mean?

Comment (keithtxw) [reply src/notification-banner/notification-banner.styles.tsx:19]: Yes it is no longer needed. Have removed

File: src/icon/types.ts [hunk 2]
```diff
@@ -1,177 +1,65 @@
...
-    | "toggle-plus"
-    | "traffic"
-    | "train"
-    | "triangle-warning"
-    | "twitter"
-    | "typography"
-    | "unlock"
+    | "tick"
+    | "ticket"
+    | "thrash-bin"
```
[the remark this answers is not in the document]
Comment (keithtxw) [reply src/icon/types.ts:57]: Haha good catch. Amended.

**Review by qroll — APPROVED** [review]

**Pull request merged**